### PR TITLE
more-less lit improvements: rearrange style changes

### PR DESCRIPTION
**Pull request by margaree**

### CODE REVIEW

File: components/more-less/more-less.js [hunk 1]
```diff
@@ -154,9 +159,16 @@ export class D2LMoreLess extends LocalizeMixin(LitElement)  {
 	}
 
 	render() {
+		var contentClasses = {
+			'more-less-content': true,
+			'more-less-transition': this.__transitionAdded
+		};
```
Comment (dlockhart) [inline components/more-less/more-less.js]: Super-nit: we can start introducing ES6 things like `let`, `const`, etc. It's finally here!

File: components/more-less/more-less.js [hunk 2]
```diff
@@ -231,7 +244,7 @@ export class D2LMoreLess extends LocalizeMixin(LitElement)  {
 		var g = (bigint >> 8) & 255;
 		var b = bigint & 255;
 
-		this.shadowRoot.querySelector('.more-less-blur').style.background =
+		this.__blurBackground =
 			'linear-gradient(rgba(' + r + ',' + g + ',' + b + ',0) 0%, rgb(' + r + ',' + g + ',' + b + ') 100%)';
```
Comment (dlockhart) [inline components/more-less/more-less.js]: Seems like a great use case for `linear-gradient(rgba(${r},${g}...`

**Comment by dbatiste** [comment]
I think there's some other `var` usage that could get switched to `const` or `let`.  :)

**Comment by margaree** [comment]
@dbatiste yeah there's definitely a few more, I didn't switch them since they weren't specific to this PR, but might as well just get it done now.

File: components/more-less/more-less.js (hunk 1 above)
Comment (svanherk) [reply components/more-less/more-less.js]: Were we unable to do this in Polymer 3?  Or we just didn't and should start?  My guess is the later...

**Comment by dbatiste** [comment]
Sounds good... looks good!

**Pull request merged**